src/main/scala/scalismo/faces/color/HSV.scala

```diff
@@ -19,34 +19,34 @@ package scalismo.faces.color
 /** HSV color value (Hue, Saturation, Value) */
 case class HSV(hue: Double, saturation: Double, value: Double) {
 
-  /** convert to RGB value */
-  def toRGB: RGB = {
+  /** convert to sRGB value */
+  def tosRGB: sRGB = {
     val hs = hue / (math.Pi / 3.0) + math.Pi
     val h: Int = hs.toInt
     val f: Double = hs - h
     val p: Double = value * (1.0 - saturation)
     val q: Double = value * (1.0 - saturation * f)
     val t: Double = value * (1.0 - saturation * (1.0 - f))
     h match {
-      case 0 => RGB(value, t, p)
-      case 1 => RGB(q, value, p)
-      case 2 => RGB(p, value, t)
-      case 3 => RGB(p, q, value)
-      case 4 => RGB(t, p, value)
-      case 5 => RGB(value, p, q)
-      case 6 => RGB(value, t, p)
-      case _ => RGB.Black // should not happen
+      case 0 => sRGB(value, t, p)
+      case 1 => sRGB(q, value, p)
+      case 2 => sRGB(p, value, t)
+      case 3 => sRGB(p, q, value)
+      case 4 => sRGB(t, p, value)
+      case 5 => sRGB(value, p, q)
+      case 6 => sRGB(value, t, p)
+      case _ => sRGB.Black // should not happen
```

Comment:
If it should not happen, then the code should fail.... maybe we should use ??? instead of a color that does not reflect the error from the outside of the function.